ci-operator/step-registry/openshift-qe/node-density/openshift-qe-node-density-ref.yaml

```diff
@@ -26,5 +26,9 @@ ref:
     default: "kube-burner-ocp.out"
     documentation: |-
       File in which we will parse to find uuid, start and end time
+  - name: GC
+    default: "true"
+    documentation: |-
+      Default is true, which means clean up the pod/resource that kube-burner ocp created, you can set it to false to keep the resouce  
```

Comment:
The default is true per [kube-burner docs](https://cloud-bulldozer.github.io/kube-burner/v1.7.6/ocp/#openshift-wrapper) so I do not think, this would be needed.

Incase a user wants to supply extra flags, this can be done via, 
`EXTRA_FLAGS` parameters
https://steps.ci.openshift.org/chain/openshift-qe-ocp-qe-perfscale-ci-tests#environment

Reply:
The e2e-benchmarking run script sets this for us so we don't need to use EXTRA_FLAGS explicitly. Having this as an environment variable will allow us at the config level to set to cleanup or not. For the loaded-upgrade tests we have we set this to false to keep the load on the cluster till after the upgrade  

https://github.com/cloud-bulldozer/e2e-benchmarking/blob/c1196a3cf0e2b34952d286fb8f33a94f19755f7b/workloads/kube-burner-ocp-wrapper/run.sh#L12

Reply:
The one thing I'm not sure of is if we need to set a default parameter in this file (I'm not sure if its required) but I feel its helpful to know what the value is without searching into e2e-benchmarkin/kube-burner